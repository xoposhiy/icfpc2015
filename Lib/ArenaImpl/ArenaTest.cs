using System.IO;
using ApprovalTests;
using Lib.Intelligence;
using Newtonsoft.Json;
using NUnit.Framework;

namespace Lib.ArenaImpl
{
    [TestFixture]
    public class ArenaTest
    {
        [Test]
        public void Test()
        {
            var arena = new Arena();
//            var solver = new PhrasesOnlySolver();
//            var solver = new NamiraOracle();
            var solver = new AzuraOracle();
            var res = arena.RunAllProblems(solver);
//            File.WriteAllText("arena.json", JsonConvert.SerializeObject(res, Formatting.Indented));
            Approvals.Verify(res);
        }
    }
}