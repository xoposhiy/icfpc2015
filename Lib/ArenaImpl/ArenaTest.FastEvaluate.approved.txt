MephalaOracle-MagicDfsFinder
TotalScore: 73223
Id	Perf	Avg	ScoreBySeeds
0	162%	8917	8917
1	122%	5136	5136
10	120%	6738	6738
13	18%	2962	2962
15	102%	6486	6486
16	237%	11876	11876
17	124%	5832	5832
19	122%	5626	5626
20	185%	8888	8888
21	37%	1466	1466
22	140%	8298	8298
23	14%	998	998